### DISCUSSION TITLE
Examples of gpttools in use

### DISCUSSION THREAD

**Discussion by JamesHWade** [JamesHWade/gpttools#1] [General]
I want to provide examples of using the package without overloading the repo size. I thought a discussion would be a good spot to do that.

## Adding Roxygen to a Function

https://user-images.githubusercontent.com/6314313/209890939-ebd7afea-7d68-40b4-b482-b3fe51485ab1.mov

## Commenting Code

https://user-images.githubusercontent.com/6314313/209890944-3d6a00fa-2d8c-4df7-8a11-f5a5ec3a1391.mov

## Converting a Script into a Function

https://user-images.githubusercontent.com/6314313/209890949-4da2bdd7-bcac-4769-9b11-7759b4abb760.mov

## Suggesting a Unit Test for a Function

https://user-images.githubusercontent.com/6314313/209890959-fca623d9-5e8e-463c-ac64-80f3db9875d9.mov

**Comment by MichelNivard** [comment]
That’s efficient, I opted for .gifs which are kind of light (<1mb), I might record a ~5 min video walkthrough that lives on YouTube and is just an embedded link in the repo

**Reply by JamesHWade** [reply]
I was fighting gifs to copy what you had done and found this article: https://www.cazzulino.com/github-pages-embed-video.html

GitHub is happy to host videos to link inside of READMEs apparently.

**Comment by MichelNivard** [comment]
Ha we have other ppl chipping in!! https://twitter.com/drlyndonwalker/status/1608395721035886594?s=46&t=_CAXiw64bRs4tJAALhQYJg

**Comment by JamesHWade** [comment]
## Documenting Data

https://user-images.githubusercontent.com/6314313/210023430-85f4cab9-9dcc-4492-a5e1-b327fd0ad81a.mov

**Comment by JamesHWade** [comment]
Here's an example of Chat in quarto

https://user-images.githubusercontent.com/6314313/210177908-7cf0f991-54f2-4f96-8bcd-3077bc8bbb25.mov

**Comment by JamesHWade** [comment]
https://user-images.githubusercontent.com/6314313/227738408-0c4c97e9-3601-4977-b8a8-ac655a185656.mov
